src/formatters/indexable-author.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Post Formatter for the indexables.
+ *
+ * @package Yoast\YoastSEO\Formatters
+ */
+
+namespace Yoast\YoastSEO\Formatters;
+
+/**
+ * Formats the term meta to indexable format.
+ */
+class Indexable_Author {
```

Comment:
I would consider renaming this class and all other classes to `Indexable_Author_Formatter`. Even though it's in the `Formatters` namespace.